applications/plugins/SofaCarving/CarvingManager.h

```diff
@@ -104,25 +97,21 @@ class SOFA_SOFACARVING_API CarvingManager : public core::behavior::BaseControlle
     
 protected:
     /// Pointer to the tool collision model
-    core::CollisionModel* m_toolCollisionModel;
+    core::CollisionModel* m_toolCollisionModel = nullptr;
 
     // Pointer to the target object collision model
     std::vector<core::CollisionModel*> m_surfaceCollisionModels;
 
     // Pointer to the scene intersection Method component
-    core::collision::Intersection* m_intersectionMethod;
+    core::collision::Intersection* m_intersectionMethod = nullptr;
     // Pointer to the scene detection Method component (Narrow phase only)
-    core::collision::NarrowPhaseDetection* m_detectionNP;
+    core::collision::NarrowPhaseDetection* m_detectionNP = nullptr;
```

Comment:
It could be a link as well so it can be explicit in the GUI.

Reply:
Done